### ISSUE TITLE
Standardize products and create a DB

### ISSUE THREAD

**Issue by oorestisime** [oorestisime/kayon-decor#36]
Store.ts served its purpose for quick launch. We should now start thinking of having one source of truth between their products listing in drive and in code to minimize discrepancies. 

* Research which DB or CMS to use (prismic, contentful, sanity, airtable, postgres db)
   - The solution should allow to define a custom object
   - Should allow to upload images and resize/crop/compress
   - Should cost too much or ideally free
   - Has a nice integration with nextjs
   - Will allow us to import existing products easily
* Figure a plan to migrate there
* This should also eliminate any custom ad-hoc code in products template (if its that chair show this, if its a basket show that)

**Comment by oorestisime** [comment]
No! 
We need something can work with. She shouldnt need to ask us to change prices descriptions etc. Add new products or whatever

**Comment by artemis-toumazi** [comment]
Yes it's better to have the autonomy for adding new products, changing the prices etc. :)

**Comment by KostasTheodoro** [comment]
A friend told me there is a way to do that, cause i asked. but since i have no idea about dbs i need to research more, i would like a timeline on this when we want it etc

**Comment by oorestisime** [comment]
Not urgent at all! Take all the time you need for your research
